src/client/stylesheets/style.less

```diff
@@ -537,20 +572,24 @@ hr.wide {
 }
 
 .label-checkbox {
-	padding: 0 .5em;
+	padding: 0 0.5em;
 	vertical-align: text-top;
 }
 
 .addToCollectionModal-body {
 	margin-right: 1em;
 	margin-left: 1em;
 	height: 18em;
-	overflow-y:scroll;
+	overflow-y: scroll;
 }
 
 .checkboxes {
-	margin-right: 0.5em !important;;
+	margin-right: 0.5em !important;
 	vertical-align: text-top;
 }
 
-
+.custom-width {
+	max-width: 200px;
+	min-width: 175px;
+	margin-top: 8px;
```

Comment:
should not we have used  bootstrap classes for the same ?
`margin-top-2` or something like that .this will increas code redablity and redundancy.

Reply:
i don't think so that it is making our code less readable. and writing a single css for margin would be better than duplicating `mt-2` for all the button classes.